### DISCUSSION TITLE
Change FontSize on CommandBarButton

### DISCUSSION THREAD

**Discussion by mgnslndh** [amwx/FluentAvalonia#388] [Ideas]
The current implementationof CommandBarButton seems to have `FontSize` hard coded to 12. How about changing it to be bound to the template parent?  That way you could override it if need be (as I have). I've not been successful in using template style override to change this value. Is it possible?

**Comment by timunie** [comment]
If it is hard coded, only option is to override entire ControlTemplate by copy & paste original one. Maybe a DynamicResouce can be introduced to adjust it, but that will require a change in source (maybe a PR provided by you will be accepted, but I cannot promise as I'm not the owner of this repo). 

For reference: https://docs.avaloniaui.net/docs/styling/troubleshooting#locally-set-properties-override-styles
